client/src/components/profile/components/Certifications.tsx

```diff
@@ -47,6 +47,8 @@ interface ICertificationProps {
 }
 
 function renderCertShow(username: string, cert: ICert): React.ReactNode {
+  // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
Better way to approach?

Reply:
I'd convert this to a standard React component.  Is it okay if I push a commit?

Reply:
Done :)